spreadsheets/functions-and-formulas/logical-functions/if.md

```diff
@@ -33,7 +33,7 @@ Example in both cases:
 
 ![if-example](https://img.enkipro.com/5561b64a6cb62ff33b21599c8aacb2d6.png)
 
-What the function outputs when its `TRUE` or `FALSE` is up to you. We could've have put "banana", "abgb315123ogua", or anything else.
+What the function outputs when its `TRUE` or `FALSE` is up to you. We could've put "banana", "abgb315123ogua", or anything else.
```

Comment:
```suggestion
What the function outputs when it's `TRUE` or `FALSE` is up to you. We could've put "banana", "abgb315123ogua", or anything else.
```

Reply:
I remember fixing that a couple of days ago. I might've forgotten to push the commit 😅

Reply:
haha, no worries